File: java/partidaFutebolMinuto.java
/*
 * Laboratório de Programação 2 - Lab 1
 *
 * @author Franklin Regis de Oliveira - 119210030
 */

import java.util.Scanner;

public class partidaFutebolMinuto{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		String[] input = sc.nextLine().split(" ");
		int golCasa = 0, golConv = 0, faltaCasa = 0, faltaConv = 0;
		while(!(input[1].equals("FIM"))){
			input = sc.nextLine().split(" ");
			if(input[1].equals("GOL")){
				if(input[2].equals("CASA")){golCasa++;}
				else{golConv++;}
			} else if(input[1].equals("FALTA")){
				if(input[2].equals("CASA")){faltaCasa++;}
				else{faltaConv++;}
			}
		}
		System.out.println(golCasa + " GOL CASA");
		System.out.println(golConv + " GOL CONVIDADO");
		System.out.println(faltaCasa + " FALTA CASA");
		System.out.println(faltaConv + " FALTA CONVIDADO");
	}
}

File: py/partidaFutebolMinuto.py
casa = []
convidado = []
while True:
    evento = input().split()
    if evento[1] == "FIM":
        break
    if evento[-1] == "CASA":
        casa.append(evento[1])
    elif(evento[-1] == "CONVIDADO"):
        convidado.append(evento[1])

print(str(casa.count("GOL")) + " GOL CASA")
print(str(convidado.count("GOL")) + " GOL CONVIDADO")
print(str(casa.count("FALTA")) + " FALTA CASA")
print(str(convidado.count("FALTA")) + " FALTA CONVIDADO")